admin/src/pages/UserSearch.vue

```diff
@@ -60,8 +72,8 @@ export default {
       searchResult: [],
       massCreation: [],
       criteria: '',
-      filterCheckedEmails: null,
-      filterDeletedUser: null,
+      filterByActivated: null,
+      filterByDeleted: null,
```

Comment:
```suggestion
      filterByActivated: false,
      filterByDeleted: false,
```

Reply:
I like that we can see explizit that we switsch between `null` and `false` or `true`.
We start with `null` here in both cases.